packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -242,6 +242,7 @@ class ClipboardCopy extends React.Component<ClipboardCopyProps, ClipboardCopySta
                     value={this.state.text as string | number}
                     id={`text-input-${id}`}
                     aria-label={textAriaLabel}
+                    dir={isCode && 'ltr'}
```

Comment:
Nit: a lot of places we'll spread an object for a case like this:

```suggestion
                    {...(isCode && { dir: 'ltr' })}
```

Both ways result in the same result, though, with the `dir` attribute not being applied at all if isCode isn't true, so not a blocker.